.drone.jsonnet

```diff
@@ -1,42 +1,87 @@
-local branches() = {
-  branch: [
-    "master",
-    "rel/*",
-    "prod/production",
-  ],
-};
+local branches() = [
+  "master",
+  "rel/*",
+  "prod/production",
+];
 
-local BuildInfo(version, limit_branches=false) = {
+local BuildInfo(version) = {
   name: "build information",
   image: "node:" + version,
   commands: [
     "node --version",
     "npm --version",
+    "yarn --version",
+    "git --version",
+    "env",
```

Comment:
remove just in case we somehow have secret envs defined globally for a pipeline instead of inside a particular pipeline step.